### PR TITLE
Main

### PR DESCRIPTION
**Pull request by matthong45**
Merging in a new empty function to test this.

### CODE REVIEW

File: Percolation.java [hunk 1]
```diff
@@ -88,7 +88,7 @@ public void open(int row, int col) {
 
     private void addingTemp()
     {
-        
+        System.out.println("Hello");
```
Comment (mjhStudent) [inline Percolation.java:91]: Did you think about doing this a different way?

File: Percolation.java [hunk 2]
```diff
@@ -88,7 +88,7 @@ public void open(int row, int col) {
...
     }
 
     private boolean tryBounds(int row, int col) {
```
Comment (mjhStudent) [inline Percolation.java:94]: This looks a bit sketch. IDK

**Review by mjhStudent — APPROVED** [review]

**Comment by mjhStudent** [comment]
LGTM

**Review by matthong45 — COMMENTED** [review]
I thought this was done.

**Pull request merged**